extensions/VisualEditor/wikia/modules/ve/ui/pages/ve.ui.WikiaParameterPage.js

```diff
@@ -16,33 +114,39 @@
  * @param {string} name Unique symbolic name of page
  * @param {Object} [config] Configuration options
  */
-ve.ui.WikiaParameterPage = function VeUiWikiaParameterPage( parameter, name, config ) {
+ve.ui.WikiaParameterPage = function VeUiWikiaParameterPage(parameter, name, config) {
```

Comment:
We need to decide what to do about the formatting. I keeped the original one as the whole VE is formatted like that.